conf/cloudformation/cloudformation.yml

```diff
@@ -76,7 +76,7 @@ Resources:
                             - content-api-alexa
                             - Ref: Stage
                             - artifact.zip
-            Description: Boilerplate
+            Description: Alexa Lambda Node/JavaScript
```

Comment:
This is more a description of what the Lambda is/does, for example:

"The Guardian's custom skill for the Amazon Echo device."